Assets/Scripts/Managers/BallManager.cs

```diff
@@ -113,6 +111,14 @@ public void SetBallToStartPosition()
         Transform startPosition = GameObject.FindWithTag("BallStartPosition").transform;
 
         StopBall(); // Stop the ball   
+        rb_ball.position = startPosition.transform.position;
+        rb_ball.rotation = startPosition.transform.rotation;
+        
+        // When setting position and rotation in the rigidbody
+        // it'll take effect during the physics update. So I believe
+        // when cinemachine tries to get the rotation to adjust the camera
+        // the rotation hasn't changed yet. So I'm setting it in the
+        // transform here as well.
         rb_ball.transform.position = startPosition.transform.position;
```

Comment:
// When setting position and rotation in the rigidbody
// it'll take effect during the physics update. So I believe
// when cinemachine tries to get the rotation to adjust the camera
// the rotation hasn't changed yet. So I'm setting it in the
// transform here as well.